components/ExperiencesList/View.js

```diff
@@ -124,19 +124,17 @@ const ExperiencesListComponent = ({ clearIdentity }) => (
       <a href="https://appliedzkp.org/" target="_blank" className="pb-14 underline" rel="noreferrer">
         Privacy & Scaling Explorations
       </a>
-      <div className="flex items-center justify-left">
-        <Link href="/identity">
-          <button className="bg-brand-yellow text-[14px] text-black py-1 px-3 rounded-full mb-16 mr-6">
-            View Semaphore ID
-          </button>
-        </Link>
-        <button
-          className="bg-brand-gray2 border-brand-yellow border-[1px] text-[14px] text-brand-yellow py-1 px-3 rounded-full mb-16"
-          onClick={clearIdentity}
-        >
-          Disconnect ID
+      <Link href="/identity">
+        <button className="w-[200px] bg-brand-yellow text-[14px] text-black py-1 px-3 rounded-full mb-10">
+          View Semaphore ID
         </button>
-      </div>
+      </Link>
+      <button
+        className="w-[200px] bg-brand-gray2 border-brand-yellow border-[1px] text-[14px] text-brand-yellow py-1 px-3 rounded-full mb-16"
```

Comment:
Try to avoid setting tailwind styles as px if you can
Like here text-[14px] is the same as text-sm, and border default is 1px so you should only need border.

This link should help out. I use it a lot